### PR TITLE
port open

### PR DESCRIPTION
**Pull request by jburel**
Clarify common connection issue in troubleshooting

https://trello.com/c/ilMDq1nx/62-openssl-telnet-commands-is-omero-responding-to-the-docs

### CODE REVIEW

File: omero/sysadmins/troubleshooting.txt [hunk 1]
```diff
@@ -136,7 +136,17 @@ following despite giving the correct username and password:
 
 This is often due to firewall misconfiguration on the machine that runs
 your OMERO server, affecting the ability of remote clients to locate
-it. Please see the :doc:`/sysadmins/server-security` page.
+it. A common issue is when port ``TCP/4064`` is not opened,
+un ``telnet server-name 4064`` to check if this is
```
Comment (mtbc) [inline omero/sysadmins/troubleshooting.txt]: "use"?

**Comment by kennethgillen** [comment]
Isn't 4063 also required to be open?

**Comment by jburel** [comment]
I can add both
but I did not want to reproduce all the info already listed in the other page (link below)
this is just to give a possible direction

**Comment by jburel** [comment]
@kennethgillen added

**Comment by kennethgillen** [comment]
@jburel - it does make sense to keep it cleaner, but best not to potentially imply that just that one port is the problem, good to show that there are 2 need opened up.

**Comment by hflynn** [comment]
Looks good on http://www.openmicroscopy.org/site/support/omero5.3-staging/sysadmins/troubleshooting.html#remote-clients-cannot-connect-to-omero-installation - I take it you are happy for this to be merged now @kennethgillen ?

**Comment by kennethgillen** [comment]
@hflynn - yes, please do - looks good.

**Pull request merged**